plugins/de.cau.cs.kieler.klighd.standalone/META-INF/MANIFEST.MF

```diff
@@ -13,6 +13,7 @@ Require-Bundle: org.eclipse.emf.ecore.xmi,
  de.cau.cs.kieler.klighd.piccolo,
  org.eclipse.elk.alg.common;resolution:=optional,
  org.eclipse.elk.alg.force;resolution:=optional,
+ org.eclipse.elk.alg.graphviz.dot,
```

Comment:
I will change it to being optional, I did not see that the others were optional as well when adding this dependency. It is required, as the LayoutMetaDataService called by the KlighdStandaloneSetup will load all ILayoutMetaDataProvider via ServiceLoader and will not be able to instanciate the GraphvizMetaDataProvider, as it has a dependency to graphviz.dot which will otherwise not be included.